modules/migration-about-cam.adoc

```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
 //
-// migration/migrating_3_4/migrating-application-workloads-3-4.adoc
-// migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
-// migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
-[id='migration-understanding-cam_{context}']
-= Understanding the Cluster Application Migration tool
+// * migration/migrating_3_4/migration-concepts-prerequisites-3-4.adoc
+// * migration/migrating_4_1_4/migration-concepts-prerequisites-4-1-4.adoc
+// * migration/migrating_4_2_4/migration-concepts-prerequisites-4-2-4.adoc
+[id='migration-about-cam_{context}']
```

Comment:
And this anchor change should be reverted. Same as with the file name changes, we shouldn't be changing anchors in published docs unless it's critical to.